src/github_provider.py

```diff
@@ -0,0 +1,58 @@
+from github import Github, PullRequest
+
+BRANCH_PREFIX = 'component-update'
+PR_TITLE_TEMPLATE = "Component {component_name} update from {old_version} to {new_version}"
+PR_BODY_TEMPLATE = """
+## what
+
+This is an auto-generated PR that updates component `{component_name}` to version `{new_version}`.
+
+| Meta               | Details            |
+|:-------------------|:-------------------|
+| **Component name** | `{component_name}` |
+| **Old Version**    | `{old_version}`    |
+| **New Version**    | `{new_version}`    |
+
+## why
+
+Use component `{component_name}` the latest available version `{new_version}` 
```

Comment:
```suggestion
[Cloud Posse](https://cloudposse.com) recommends upgrading Terraform components regularly to maintain a secure, efficient, and well-supported infrastructure management system. In doing so, you'll benefit from the latest features, improved compatibility, easier upgrades, and ongoing support while mitigating potential risks and meeting compliance requirements.

- **Enhanced Security**: Updating to the latest version ensures you have the most up-to-date security patches and vulnerability fixes, safeguarding your infrastructure against potential threats.
- **Improved Performance**: Newer versions of Terraform components often come with performance optimizations, enabling your infrastructure to run more efficiently and use resources more effectively.
- **Access to New Features**: Regularly updating ensures access to the latest features and improvements, allowing you to take advantage of new functionality that can help streamline your infrastructure management process.
- **Better Compatibility:** Staying current with Terraform component updates ensures that you maintain compatibility with other components that may also be updating. This can prevent integration issues and ensure smooth operation within your technology stack.
- **Easier Upgrades:** By updating components regularly, you can avoid the challenges of upgrading multiple versions at once, which can be time-consuming and potentially introduce breaking changes. Incremental updates make it easier to identify and resolve issues as they arise.
- **Ongoing Support:** Updated components are more likely to receive continued support from the developer community, including bug fixes, security patches, and new feature development. Falling too far behind can leave your infrastructure unsupported and more vulnerable to issues.
- **Compliance Requirements:** In some cases, regularly updating your Terraform components may be necessary to meet regulatory or industry compliance requirements, ensuring that your infrastructure adheres to specific standards.
```